TEP002_model_hdf5.rst

```diff
@@ -0,0 +1,62 @@
+TEP002: Meaningful title
```

Comment:
TEP0002: TARDIS hdf5 data storage capabilities

something like this, since it is not exclusive to the model any more?